openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py

```diff
@@ -549,85 +530,77 @@ def test_get_applicant_info_fieldset(user_application, organization):
     assert expected_fieldset == actual_fieldset
 
 
-@pytest.mark.parametrize('resume, cover_letter_file, cover_letter_text, expected_fieldset_title, expected_fields', [
-    (TEST_RESUME, TEST_COVER_LETTER_FILE, None, RESUME_AND_COVER_LETTER, [RESUME, COVER_LETTER_FILE]),
-    (TEST_RESUME, False, TEST_COVER_LETTER_TEXT, RESUME_AND_COVER_LETTER, [RESUME]),
-    (TEST_RESUME, False, False, RESUME_ONLY, [RESUME]),
-    (False, TEST_COVER_LETTER_FILE, False, COVER_LETTER_ONLY, [COVER_LETTER_FILE]),
-    (False, False, TEST_COVER_LETTER_TEXT, COVER_LETTER_ONLY, [])
+@pytest.mark.parametrize('resume, interest_in_program, expected_fieldset_title, expected_fields', [
+    (TEST_RESUME, None, RESUME_ONLY, [RESUME]),
+    (TEST_RESUME, TEST_INTEREST_IN_PROGRAM, RESUME_AND_INTEREST, [RESUME]),
+    (False, TEST_INTEREST_IN_PROGRAM, INTEREST_ONLY, [])
 ], ids=[
-    'with_resume_and_cover_letter_file',
-    'with_resume_and_cover_letter_text',
     'with_resume_only',
-    'with_cover_letter_file_only',
-    'with_cover_letter_text_only'
+    'with_resume_and_interest',
+    'with_interest_only'
 ])
 @pytest.mark.django_db
-@mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_resume_cover_letter_display_fields')
-def test_get_fieldset_for_resume_cover_letter(
-    mock__get_resume_cover_letter_display_fields,
+@mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_resume_and_interest_display_fields')
+def test_get_fieldset_for_resume(
```

Comment:
name incomplete.

"test_get_fieldset_for_resume_and_interest"

The convention is to add 'test_' as a prefix to the function being tested.